<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_DA_Edit_Delete</class>
 <widget class="QWidget" name="Page_DA_Edit_Delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Left">
       <item>
        <widget class="QGroupBox" name="groupBox_known_DAs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>20</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Bekannte Daueraufträge</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_CurrentDAs">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <column>
             <property name="text">
              <string>Kto-Nr.</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>BLZ</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Begünstigter</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Betrag</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Akt_Del">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_DA_Delete">
              <property name="toolTip">
               <string>Löscht den ausgewählten Dauerauftrag</string>
              </property>
              <property name="statusTip">
               <string>Auftrag zum Löschen in den Ausgangskorb stellen</string>
              </property>
              <property name="text">
               <string>Löschen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_DA_Aktualisieren">
              <property name="toolTip">
               <string>Aktualisert die Liste der Daueraufträge</string>
              </property>
              <property name="statusTip">
               <string>Stellt einen Aktualisierungs-Aufrag für die Daueraufträge in den Ausgangskorb</string>
              </property>
              <property name="text">
               <string>Aktualisieren</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_right">
       <property name="spacing">
        <number>4</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Exec_Revert">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Revert">
       <property name="text">
        <string>Rückgänging</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Execute">
       <property name="text">
        <string>Ausführen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
